book/src/pong_tutorial/pong_tutorial_03.md

```diff
@@ -28,16 +28,17 @@ project, called `bindings_config.ron`:
 ```ron,ignore
 (
   axes: {
-    "left_paddle": (pos: Key(W), neg: Key(S)),
-    "right_paddle": (pos: Key(Up), neg: Key(Down)),
+    "left_paddle": Emulated(pos: Key(W), neg: Key(S)),
+    "right_paddle": Emulated(pos: Key(Up), neg: Key(Down)),
   },
   actions: {},
 )
 ```
 
 In Amethyst, inputs can be either scalar inputs (a button that is either
-pressed or not), or axes (a range that relates two buttons as opposite
-ends of a range). In this file, we're creating two axes: W and S will move the
+pressed or not), or axes (a range that represents analog controller stick or
```

Comment:
that represents an analog

Reply:
That's right, axes are analog when bound to analog stick, returning values from `-1` to `1`.

Reply:
no, you are missing a "an" word

Reply:
Don't forget many gamepads and such now have 'buttons' that have a depress amount, I.E. they are just axis in button form.  Honestly every button and axis and all should just be a float from 1.0 to 1.1, just for ones that are binary they just go between the two values 0.0 and 1.0.

Reply:
@OvermindDL1 This can be treated like an axis. In fact, that's how the "trigger" button works right now on everything I tested. Currently there is a limited set of axes in the `ControllerAxis` enum, as this is the set supported by SDL, but this list might become dynamic later. I would prefer to keep it simple for now.